tests/searchimagescreativetests.js

```diff
@@ -63,10 +63,10 @@ test.beforeEach(t=>{
             .reply(200, {response : "response"})
             .get("/v3/search/images/creative")
             .query({ "page": 3, "phrase": "cat" })
-            .reply(200, {response : "response"})
```

Comment:
Every combination here just returns `{response : "response"}` so it seems that the tests have a high chance of false passes.

These changes highlight the rough basics of what I think needs to happen.

Reply:
@ChrisSimmons  Any plans to continue this branch or should we close this and delete the branch?